CMakeLists.txt

```diff
@@ -1,5 +1,41 @@
-cmake_minimum_required(VERSION 3.10.2 FATAL_ERROR)
+# ==== Define cmake build policies that affect compilation and linkage default behaviors
+#
+# Set the ITK_NEWEST_VALIDIDATED_POLICIES_VERSION string to the newest cmake version
```

Comment:
VALID**ID**ATED -> VALIDATED

This misspelling is consistent, and each instance should be changed.

Reply:
Dzenanz,

Thank you.  I'm sorry about so many spelling/character transposition/repetition issues.  My brain just does not process that information correctly :(.  I don't see those errors :(.  

Hans

Reply:
It looked suspicious at first, but then I thought it was OK. And just when I was about to click "Approve", I spotted what was bugging me 😄